apps/AmplifyDemo/README.md

```diff
@@ -0,0 +1,124 @@
+    ? Distribution Directory Path: dist 
+    ? Build Command: npm run-script build
+    ? Start Command: npm run-script start 
+    ? Do you want to use an AWS profile? Yes 
+    ? Please choose the profile you want to use [Your AWS Profile] <- Choose the AWS Profile configured in the previous step
+    ```
+
+### Generate backend resources using Amplify CLI
+
+Amplify provides a CLI command to generate a predefined group of backend resources, referred to as a “category”. In this section, you’ll be adding the `api` and `function` categories to our application backend. 
+
+First, you’ll add the `function` category using the Amplify CLI. This command will create a directory inside `./amplify/backend/{functionName}` where you can define the implementation of a Lambda function. Amplify will also generate the AWS CloudFormation template required in order to deploy the Lambda function to your AWS account.
+
+Next, you’ll add the `api` category. Amplify will automatically create a GraphQL-based API using AWS AppSync behind the scenes. Again, it will generate all of the CloudFormation templates to create these resources in your AWS account. You’ll create a `schema.graphql` file that will define which APIs are required - and what their definitions are.
+
+Lastly, you’ll use the Amplify CLI to push these configuration up to your AWS account, and Amplify will read the configurations for each category that you added and generate the corresponding resources in your AWS account. Note that none of the changes will be visible in your AWS account until you complete this step. Any further updates to your local Amplify environment can be updated in your AWS account by repeating this `push` step. 
+
+#####1. Create a Lambda in your local Amplify environment using the Amplify CLI. 
+This Lambda will act as a resolver for your APIs, meaning that AWS AppSync will send the GraphQL API requests to this Lambda. The Lambda contains code to handle different GraphQL API requests and return a response back to your client application. Run the command below and answer the prompts as shown below: 
+    IMPORTANT: Make sure to use the name `reactSampleLambda` for the purpose of this demo. You may choose a different name, but make sure the name matches the name of the lambda function defined in `schema.graphql`.
+
+    ```
+    $ amplify add function
+
+    ? Select which capability you want to add: Lambda function (serverless function)
+    ? Provide an AWS Lambda function name: reactSampleLambda 
+    ? Choose the runtime that you want to use: NodeJS
+    ? Choose the function template that you want to use: Hello World
+    ? Do you want to configure advanced settings? No
+    ? Do you want to edit the local lambda function now? Yes
+    ```
+
+    When asked “Do you want to edit the local lambda function now?,” answer ‘Yes’ and replace the `./amplify/backend/function/reactSampleLambda/src/index.js` with the provided code in `./lambda/index.js`.
```

Comment:
Fix markdown formatting. The above code block includes this question.

![image](https://user-images.githubusercontent.com/36240708/124990205-e744eb80-dff4-11eb-96f3-8f49fe8e56bf.png)